### ISSUE TITLE
en_pytt_xlnetbasecased_lg not found

### ISSUE THREAD

**Issue by boxabirds** [explosion/spacy-pytorch-transformers#16]
## How to reproduce the behaviour
Run [this notebook](https://colab.research.google.com/drive/1ItRSEoukjI3wYq5pHOfd6ZhLj2Oc6Gzc#scrollTo=nIewjdRErzFz) 

```
!python -m spacy download en_pytt_xlnetbasecased_lg
…
Building wheels for collected packages: en-pytt-xlnetbasecased-lg
  Building wheel for en-pytt-xlnetbasecased-lg (setup.py) ... done
  Created wheel for en-pytt-xlnetbasecased-lg: filename=en_pytt_xlnetbasecased_lg-2.1.0-cp36-none-any.whl size=433908237 sha256=002075047539e01a91204dece545d44b6f89e392f4424cb2bf7922c925909172
  Stored in directory: /tmp/pip-ephem-wheel-cache-vhhkwobw/wheels/7f/5a/dd/015d2bd7860df17954162993bf20100f1fa051184d3588eec6
Successfully built en-pytt-xlnetbasecased-lg
Installing collected packages: en-pytt-xlnetbasecased-lg
Successfully installed en-pytt-xlnetbasecased-lg-2.1.0
✔ Download and installation successful
You can now load the model via spacy.load('en_pytt_xlnetbasecased_lg')
…
nlp = spacy.load("en_pytt_xlnetbasecased_lg")
OSError: [E050] Can't find model 'en_pytt_xlnetbasecased_lg'. It doesn't seem to be a shortcut link, a Python package or a valid path to a data directory.
```

## Your Environment

* Operating System: Google Colab VM
* Python Version Used: Python 3.6
* spaCy Version Used: 2.1.7
* Environment Information: Google Colab

**Comment by BrambleXu** [comment]
I download en_pytt_xlnetbasecased_lg in colab and link it successfully. But still can not load the model.
```
!python -m spacy download en_pytt_xlnetbasecased_lg

Building wheels for collected packages: en-pytt-xlnetbasecased-lg
  Building wheel for en-pytt-xlnetbasecased-lg (setup.py) ... done
  Created wheel for en-pytt-xlnetbasecased-lg: filename=en_pytt_xlnetbasecased_lg-2.1.0-cp36-none-any.whl size=433908237 sha256=6572e7294f9db296981d6acc69572e037fbb1f8723f98309b293a676d6907b4f
  Stored in directory: /tmp/pip-ephem-wheel-cache-134h8tb3/wheels/7f/5a/dd/015d2bd7860df17954162993bf20100f1fa051184d3588eec6
Successfully built en-pytt-xlnetbasecased-lg
Installing collected packages: en-pytt-xlnetbasecased-lg
Successfully installed en-pytt-xlnetbasecased-lg-2.1.0
✔ Download and installation successful
You can now load the model via spacy.load('en_pytt_xlnetbasecased_lg')
```

```
!python -m spacy link /usr/local/lib/python3.6/dist-packages/en_pytt_xlnetbasecased_lg en_pytt_xlnetbasecased_lg

✔ Linking successful
/usr/local/lib/python3.6/dist-packages/en_pytt_xlnetbasecased_lg -->
/usr/local/lib/python3.6/dist-packages/spacy/data/en_pytt_xlnetbasecased_lg
You can now load the model via spacy.load('en_pytt_xlnetbasecased_lg')
```

```
spacy.load('en_pytt_xlnetbasecased_lg')

---------------------------------------------------------------------------
ModuleNotFoundError                       Traceback (most recent call last)
/usr/local/lib/python3.6/dist-packages/spacy/util.py in get_lang_class(lang)
     69         try:
---> 70             module = importlib.import_module(".lang.%s" % lang, "spacy")
     71         except ImportError as err:

11 frames
ModuleNotFoundError: No module named 'spacy.lang.pytt'

During handling of the above exception, another exception occurred:

ImportError                               Traceback (most recent call last)
/usr/local/lib/python3.6/dist-packages/spacy/util.py in get_lang_class(lang)
     70             module = importlib.import_module(".lang.%s" % lang, "spacy")
     71         except ImportError as err:
---> 72             raise ImportError(Errors.E048.format(lang=lang, err=err))
     73         LANGUAGES[lang] = getattr(module, module.__all__[0])
     74     return LANGUAGES[lang]

ImportError: [E048] Can't import language pytt from spacy.lang: No module named 'spacy.lang.pytt'

---------------------------------------------------------------------------
NOTE: If your import is failing due to a missing package, you can
manually install dependencies using either !pip or !apt.

To view examples of installing some common dependencies, click the
"Open Examples" button below.

**Comment by boxabirds** [comment]
Interesting now I can't even get BERT to install in the same notebook as before with a fresh VM. I can't seem to reopen this issue so I'll create another one

**Comment by ines** [comment]
@boxabirds Feel free to open a new issue 🙂 Btw, we just released spaCy v2.1.8 and I think I finally found a (hopefully) stable fix to allow loading an installed model package in the same session 🎉 See: explosion/spaCy#4090.
